requirements/edx/github.txt

```diff
@@ -39,3 +39,6 @@ git+https://github.com/mitocw/django-cas.git@60a5b8e5a62e63e0d5d224a87f0b489201a
 -e git+https://github.com/edx/edx-milestones.git@547f2250ee49e73ce8d7ff4e78ecf1b049892510#egg=edx-milestones
 -e git+https://github.com/edx/edx-search.git@264bb3317f98e9cb22b932aa11b89d0651fd741c#egg=edx-search
 git+https://github.com/edx/edx-lint.git@e0e73bcc7b3b7ed632a1db6e3b32b0a249721261#egg=edx_lint
+-e git+https://github.com/edx/xblock-utils.git@349d6e05dbd553e1f18d3ad1f7ca02c0497f39d7#egg=xblock-utils
```

Comment:
@martynjames @nedbat Our pending review is only with additional changes which aren't yet in master on that repository: https://github.com/edx/xblock-utils/pull/5

For the review of the current master of xblock-utils, I assumed it was done as part of the google-drive code review since you had created https://github.com/edx/xblock-utils/pull/6 ?